dev/com.ibm.ws.beanvalidation.v20.cdi/src/com/ibm/ws/beanvalidation/v20/cdi/internal/BvalManagedObjectBuilderImpl.java

```diff
@@ -285,4 +346,30 @@ private Class<?> loadClass(String className, ClassLoader appClassLoader) {
             throw new ValidationException(nls.getString("BVKEY_UNABLE_TO_CREATE_VALIDATION_FACTORY", "BVKEY_UNABLE_TO_CREATE_VALIDATION_FACTORY"), e);
         }
     }
+
+    private void setVersion() {
+        provisionerService.getInstalledFeatures().forEach(feature -> {
+            String subString = null;
+            if (feature.startsWith("beanValidation-")) {
```

Comment:
technically feature names are not case sensitive  ... you might get away with it here since you're only processing what is coming back from the installed features??

Reply:
This is not the normal way that you should be tracking version of a feature.  This approach is definitely the anti-pattern.